testcrate/src/lib.rs

```diff
@@ -41,3 +42,16 @@ fn lua_works() {
         assert_eq!(version, "Lua 5.4".as_bytes());
     }
 }
+
+#[test]
+fn unicode_identifiers() {
+    unsafe {
+        let state = luaL_newstate();
+        let code = "local 😀 = 0\n";
+        let ret = luaL_loadstring(state, code.as_ptr().cast());
```

Comment:
`luaL_loadstring` accepts zero-terminated string, whereas supplied string does not have terminating `\0`. Could you add please?

Reply:
Okey I fixed it.